contributing/contributing_to_datacommons.md

```diff
@@ -4,12 +4,20 @@ title: Contributing to Data Commons
 nav_order: 6
 has_children: true
 ---
-# How to Contribute
+# Contribute to Data Commons!
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+Data Commons contributors:
 
-## Contributor License Agreement
+- Contribute data
+- Create tools
+- Share analysis
+- Write documentation
```

Comment:
Maybe we can add hyperlinks to the corresponding sections below?

Reply:
That's a great idea! I've incorporated it.